db/schema.rb

```diff
@@ -10,15 +10,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_09_23_153522) do
+ActiveRecord::Schema.define(version: 2020_09_23_154527) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_enum :animal_sex, [
-      "unknown",
-      "male",
-      "female",
+    "unknown",
+    "male",
+    "female",
```

Comment:
Weird that this ended up in the diff. Maybe it has to do with the version of Postgres your running locally? Ive seen that be the cause of weird diffs in `schema.rb` in other projects in the past.